internal/kafka/internal/migrations/20220621100000_add_kafka_billing_model.go

```diff
@@ -0,0 +1,22 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+func addKafkaBillingModel() *gormigrate.Migration {
+	type KafkaRequest struct {
+		BillingModel string `json:"billing_model"`
+	}
+
+	return &gormigrate.Migration{
+		ID: "20220621100000",
+		Migrate: func(tx *gorm.DB) error {
+			return tx.AutoMigrate(&KafkaRequest{})
```

Comment:
Yes, that's the intention, same as with the other params. When the value is empty, the billing model is not validated and the previous logic applies.

Reply:
ok. thanks

Reply:
Is it possible to pre-fill the value of  `billing_model` column for existing kafkas based on other info that we current store in the database?  This will be better than leaving this as empty for all existing kafkas.

Reply:
Specifically I was thinking of having another query that updates all existing kafkas billing_model to `marketplace` if https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/0844d0bbbc615c5a17105c5a2878236bee686838/internal/kafka/internal/migrations/20220524140500_add_kafka_marketplace_cloud_account_id.go#L10-L11 if set. 
Otherwise, set it to `standard`

What do you think @pb82 @pawelpaszki ?